src/config.py

```diff
@@ -651,4 +651,35 @@ def from_network(network: Network) -> AccountingConfig:
         )
 
 
+@dataclass(frozen=True)
+class OverdraftConfig:
+    """Full configuration for the overdrafts management contract"""
+
+    contract_address: str
+
+    @staticmethod
+    def from_env() -> OverdraftConfig:
```

Comment:
All other classes in `config.py` that only need network information have an initializer that is `from_network(network: Network)`. So, let's use that convention here as well